.travis.yml

```diff
@@ -5,10 +5,7 @@ node_js:
 env:
   global:
   - CF_API=https://api.fr.cloud.gov
-  - CF_USERNAME=gsa-18f-federalist_deployer
   - CF_ORGANIZATION=gsa-18f-federalist
```

Comment:
Following up from the federalist pull comment, what we have here in federalist-builder is what is intended.